bulksendemail.js

```diff
@@ -83,27 +88,21 @@ var tjutilitysendemail = {
 			// Remove below line removeclass it is temp added
 			jQuery("div").find('.tjlms-wrapper').removeClass("tjBs3");
 
-			var modelEmail = '<div id="bulkEmailModal" class="emailModal modal fade" role="dialog">';
-					modelEmail += '<div class="modal-dialog">';
-						modelEmail += '<div class="modal-content">';
-						modelEmail += '<div id="preload"><img src="http://i.imgur.com/KUJoe.gif"></div>';
+			var modelEmail = '<div id="bulkEmailModal" class="custom-modal modal fade" role="dialog">';
```

Comment:
Can we not create php joomla layouts and use those to generate HTML

We can use separate layouts for bs2, bs3
- use bs2 in backend
- use bs3 in frontend

Maybe add config in plugin for what to use for frontend: bs2/3

@twsvaishali @coolbung

Reply:
Can be done later in required